english/forest/forest.md

```diff
@@ -0,0 +1,24 @@
+An overwhelming pain echoes through your head. You feel the taste of dirt in
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.